content/en/real_user_monitoring/guide/best-practices-tracing-native-ios-android-apps.md

```diff
@@ -0,0 +1,167 @@
+- Understand how different methods and components interact
+- Break down performance bottlenecks
+- Gain actionable insights into app behavior
+
+Here's how to manually instrument a span in your mobile app using the Trace SDK:
+
+{{< tabs >}}
+{{% tab "iOS" %}}
+```swift
+let span = tracer.startSpan(operationName: "<span_name>")
+// ... code to measure ...
+span.finish()
+```
+See [more iOS examples][1].
+
+[1]: /tracing/trace_collection/automatic_instrumentation/dd_libraries/ios?tab=swiftpackagemanagerspm
+
+{{% /tab %}}
+{{% tab "Android" %}}
+
+```kotlin
+val span = tracer.buildSpan("<span_name>").start()
+// ... code to measure ...
+span.finish()
+```
+See [more Android examples][1].
+
+[1]: /tracing/trace_collection/automatic_instrumentation/dd_libraries/android/?tab=kotlin
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Sampling
+
+Sampling in native mobile tracing controls which spans and traces are ingested in Datadog, helping you balance visibility with data volume. When you manually instrument spans in your app, the ones that are sampled in go through the APM retention filters which determine which ones end up being displayed for analysis in the Datadog interface.
+
+### Sampling parameters
+
+The following sampling rate parameters control different aspects of data collection:
+
+{{< tabs >}}
+{{% tab "iOS" %}}
+
+| Parameter                                   | Description                                                      |
+|---------------------------------------------|------------------------------------------------------------------|
+| `Trace.Configuration.sampleRate`                          | Controls the percentage of manually instrumented spans collected by the tracer.|
+| `urlSessionTracking.firstPartyHostsTracing.sampleRate` | Controls the percentage of network requests traced for first-party hosts. |
+
+**Note**: `RUM.sessionSampleRate` controls RUM session sampling and does not affect trace sampling rates. When using the Trace SDK independently of RUM, trace sampling is controlled only by the parameters listed above.
+
+{{% /tab %}}
+{{% tab "Android" %}}
+
+| Parameter                                   | Description                                                      |
+|---------------------------------------------|------------------------------------------------------------------|
+| `AndroidTracer.Builder.setSampleRate`                          | Controls the percentage of manually instrumented spans collected by the tracer.|
+| `DatadogInterceptor.Builder.setTraceSampler` | Controls the percentage of network requests traced for first-party hosts. |
+
+**Note**: `RUM.sessionSampleRate` controls RUM session sampling and does not affect trace sampling rates. When using the Trace SDK independently of RUM, trace sampling is controlled only by the parameters listed above.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### How sampling works
```

Comment:
Ah, I see my question from above was answered in this section! I think it might make more sense to put this section first, and then use the section about distributed trace sampling to introduce the table above, or maybe link down to the parameter section. That way, the progressive disclosure part is a little neater - first you learn about the kinds of sampling there are, and then you learn more about how to implement them.